src/docs/020_tutorial/070_publishToConfluence.adoc

```diff
@@ -26,38 +26,44 @@ In this tutorial, you'll learn how to publish the arc42 template to a Confluence
 
 If you  have completed instructions http://doctoolchain.org/docToolchain/v2.0.x/020_tutorial/010_Install.html[install docToolchain] and http://doctoolchain.org/docToolchain/v2.0.x/020_tutorial/020_arc42.html[get the arc42 template] you can skip this part.
 
-NOTE: For this tutorial, we assume that you work with a MacOs/Linux-based system.
+NOTE: For this tutorial, we assume that you work with a macOS/Linux-based system.
 
 
 . In the Terminal, type the following:
 +
-[source, bash]
+[source, console]
 ----
-mkdir publishToConfluenceDemo
-cd publishToConfluenceDemo
-curl -Lo dtcw doctoolchain.org/dtcw`
+$ mkdir publishToConfluenceDemo
```

Comment:
I am not so sure about this change. The prefix makes it look more like a console, but what about copy & paste? I guess it will be broken.

We could move the $ to the css as a prefix.

What do you think?

Reply:
I've taken a look at the [AsciiDoctor documentation](https://docs.asciidoctor.org/asciidoctor/latest/cli/) itself. They use a simple indented literal block for single commands with the `$` included. But on the rendered page this is transformed into a code listing block highlighted with highlight.js and language shell and the `$` is somehow removed from the line when copied. I'm not sure how that is achieved.

source:
```asciidoc
 $ asciidoctor --version
```

rendered page:
```html
<code class="language-console hljs shell" data-lang="console">
  <span class="hljs-meta">$</span> asciidoctor --version
</code>
```

I would prefer to have the `$` being shown, as you pointed out it makes it stand out as a command being entered. I had hoped the switch from `bash` being a shell script to `console` would already do the trick.

And after some more research this seems to be achieved via the CSS proper `user-select: none` on the `.hljs-meta` elements.

The best way forward for me seem to be to mark console commands as `[source, console]` and add some CSS properties.

Reply:
And just now I realized that docToolchain does not use highlight.js for code highlighting on the microsite but google-code-prettify. Which is seems to be no longer supported https://github.com/googlearchive/code-prettify.

So now I don't know how to achieve having the `$` and not breaking the copy&paste flow.

Reply:
ok. let's switch to highlightjs...

Reply:
I tried to implement highlightjs in this branch and pushed the result, but I don't know where they where pushed to :-o

So I created a new issue and PR: https://github.com/docToolchain/docToolchain/pull/1238